### DISCUSSION TITLE
[Feature Request - Traces Visualization] Copy/Paste trace or span  details as JSON and/or Markdown

### DISCUSSION THREAD

**Discussion by cyrille-leclerc** [grafana/grafana#61370] [Observability]
When troubleshooting a performance problem, I want to capture the details of a span or a trace copying all its details (incl attributes, resource attributes, and events) as JSON and/or as Markdown.
JSON would be my preference.

I would like this GUI to be consistent with:

* https://github.com/grafana/grafana/discussions/61371

![image](https://user-images.githubusercontent.com/459691/212074638-4f85f2b3-863e-40bd-889e-cfd60aac2800.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82814. Feel free to continue the discussion around this there. Thank you!
